writable.go

```diff
@@ -18,6 +18,13 @@ func BytesWritable(b []byte) []byte {
 	return b[4:]
 }
 
+func PutBytesWritable(raw []byte) []byte {
```

Comment:
We're missing `PutText`, `PutIntWritable`,  etc. Also, for consistency with `encoding/binary`'s `Put*` methods, I think it'd be better to pass in the byte slice.